packages/ui/src/lib/screen/EmptyScreen.svelte

```diff
@@ -1,26 +1,48 @@
 <script lang="ts">
 import { faPaste } from '@fortawesome/free-solid-svg-icons';
-import { createEventDispatcher, onMount } from 'svelte';
+import { createEventDispatcher, onMount, type Snippet } from 'svelte';
 import Fa from 'svelte-fa';
 
 import Button from '../button/Button.svelte';
 import { isFontAwesomeIcon } from '../utils/icon-utils';
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export let icon: any;
-export let title = 'No title';
-export let message = 'Message';
-export let detail = '';
-export let commandline = '';
-export let hidden = false;
+interface Props {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  icon: any;
+  title?: string;
+  message?: string;
+  detail?: string;
+  commandline?: string;
+  hidden?: boolean;
+  class?: string;
+  style?: string;
+  'aria-label'?: string;
+  upperContent?: Snippet;
+  children?: Snippet;
+  onClick?: (text: string) => void;
```

Comment:
onclick ? let's be consistent with how svelte do the naming ? (I don't like it, but maybe better to have a single way)

Reply:
if the `onClick` is for backward compatiblity, let's keep it

Reply:
for backward compliance, we need to dispatch an event which is already the case

but yes, the callback should be named with the same logic than svelte --> onclick